neuromllite/NetworkGenerator.py

```diff
@@ -450,9 +450,9 @@ def generate_neuroml2_from_network(nl_model,
     
     if simulation is not None:
         if simulation.dt is not None:
-            print nml_doc.networks[0].properties.append(neuroml.Property('recommended_dt_ms',simulation.dt))
```

Comment:
Thanks for spotting this @sanjayankur31, but that print (and below) was actually a typo... Removed in master.